server/src/test/scala/org/elasticmq/server/config/ElasticMQServerConfigTest.scala

```diff
@@ -13,15 +13,15 @@ class ElasticMQServerConfigTest extends AnyFunSuite with Matchers with OptionVal
 
   test("load the test config") {
```

Comment:
we need a test here as well, that the configuration is properly read. Additionally the persisted queue config should take precedence over config for the same queues, but defined in the config file